### DISCUSSION TITLE
Could not find a part of the path Error

### DISCUSSION THREAD

**Discussion by jimjano** [community/community#160569] [Actions] [answered]
All,

I am just getting into GitHub actions.  I've already set up a self-hosted on a windows server.  Trying a simple action and it's erroring out in the actions/checkout@v2 section.  Here is the relevant info:

`C:\windows\System32\WindowsPowerShell\v1.0\powershell.exe -NoLogo -Sta -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "$ErrorActionPreference = 'Stop' ; try { Add-Type -AssemblyName System.IO.Compression.FileSystem } catch { } ; [System.IO.Compression.ZipFile]::ExtractToDirectory('C:\actions-runner\_work\Project1\Project1\6574178c-da1d-4ac9-ade9-70735c3699db.tar.gz', 'C:\actions-runner\_work\Project1\Project1\6574178c-da1d-4ac9-ade9-70735c3699db')"
Exception calling "ExtractToDirectory" with "2" argument(s): "Could not find a part of the path 'C:\actions-runner\_wor
k\Project1\Project1\6574178c-da1d-4ac9-ade9-70735c3699db\Org-Project1-68f6527b77d2d6130f2c721a0d6bc9318b6b7f2c\Project1App\Controllers\AppointmentController.cs
'."
At line:1 char:111
+ ... catch { } ; [System.IO.Compression.ZipFile]::ExtractToDirectory('C:\a ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : DirectoryNotFoundException`

I've googled and stackoverflow'd and couldn't find any info.  Does anyone have any thoughts?

Thanks!

**Comment by jimjano** [comment, marked as answer]
Figured it out...path was too long.   Needed to edit the registry:

HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem and set LongPathsEnabled to 1

**Reply by inzi** [reply]
Thanks for this.

**Reply by adamewozniak** [reply]
Ditto ^

**Reply by Danja30** [reply]
@jimjano Hai a'm nee het!!

**Reply by james-perfectserve** [reply]
powershell command is `Set-ItemProperty 'HKLM:\System\CurrentControlSet\Control\FileSystem' -Name 'LongPathsEnabled' -value 1`

**Reply by e-simo** [reply]
To avoid editing the repository, you can check this: https://www.microfocus.com/documentation/filr/filr-4/filr-desktop/t47bx2ogpfz7.html
worked for me
